disnake/types/gateway.py

```diff
@@ -241,3 +257,352 @@ class VoiceResumeData(TypedDict):
 class VoiceResumeCommand(TypedDict):
     op: Literal[7]
     d: VoiceResumeData
+
+
+# events
```

Comment:
This comment looks a little awkward 😄

Reply:
Hmm, good point 😅 Tried to improve this a bit in 470156227e22ba706e48479fcd9bdaf5d3fb13f6.